build.gradle

```diff
@@ -80,3 +84,39 @@ subprojects {
 }
 
 apply from: file('config/gradle/dependencyGraph.gradle')
+
```

Comment:
We can move the code to another file and apply it from there, to keep `build.gradle` less crowded.

Reply:
Yes of course!